packages/communication-react/CHANGELOG.md

```diff
@@ -11,7 +11,20 @@ Thu, 22 Jul 2021 17:42:41 GMT
 
 ### Changes
 
-- Bump @internal/chat-stateful-client to v1.0.0-beta.3 ([PR #554](https://github.com/azure/communication-ui-library/pull/554) by 2684369+JamesBurnside@users.noreply.github.com)
+* [Breaking change] Call Composite and Chat Composite adapters now take in a `CommunicationTokenCredential` in the constructor instead of the token
+
+* [Breaking change] Call Composite and Chat Composite adapters take in an object containing all of the parameters instead of passing in the parameters individually
+
+* Fixing the sendbox component where the padding would overlap with the icon
+
+* Fixing the local preview overlapping the call control bar
+
+* Setting minHeight on the call composite media gallery
+
+* Adding a small fix for supporting typescript < 4.1
+
```

Comment:
Some extra things we could consider adding:
react-components:
- React hook useTheme added to ACS library.
- Fix truncation from MessageThread system message.
- Add default call control bar button for creating custom call control buttons
- Fix cursor on control button labels.
- Add identifier injection mechanism in components for e2e testing
- Exporting locales as separate entites.
- Localized strings for SendBox and MessageStatusIndicator
- Fix MessageThread hook related console errors.
- Add an ErrorBar component
react-composites:
- Make participant pane in the chat composite optional. Disabled by default.
- Chat composite topic heading is now optional. Disabled by default.
- Add ErrorBar in ChatComposite behind a feature flag
- Bugfix to prevent options menu from getting hidden every time a participant joins or leaves
- Drop duplicate thread status banner

Reply:
Also: Publish `js` files to use composites outside of a react app as part of the GitHub release

Reply:
Oh right. 

I'll add those. I think for now till we're ready to share more about the ErrorBar and Localization we will exclude them from these notes for now.